import_obfuscation.go

```diff
@@ -44,28 +48,59 @@ type privateImports struct {
 	privateNames []string
 }
 
-func obfuscateImports(objPath, importCfgPath string) (map[string]string, error) {
+func appendPrivateNameMap(nameMap map[string]string, pkg string, packageDirectory string) error {
+	mapFile := filepath.Join(packageDirectory, garbleMapFile)
+	data, err := ioutil.ReadFile(mapFile)
+	if errors.Is(err, os.ErrNotExist) {
+		return nil
+	}
+
+	if err != nil {
+		return err
+	}
+
+	var localMap map[string]string
+	err = json.Unmarshal(data, &localMap)
+	if err != nil {
+		return err
+	}
+
+	for oldName, newName := range localMap {
+		nameMap[pkg+"."+oldName] = newName
+	}
+
+	return nil
+}
+
+func obfuscateImports(objPath, importCfgPath string) (map[string]string, map[string]string, error) {
```

Comment:
name the two returned maps; you deduplicate the type, and document what they are.